### PR TITLE
fixes for fontawesome pencil icon

### PR DESCRIPTION
**Pull request by bradymiller**
inspired by @adad20

### CODE REVIEW

**Comment by adad20** [comment]
hi @bradymiller you should have told me in the PR I should have fix this. Sorry for the incomplete work 😅

**Comment by bradymiller** [comment]
I gotta get a commit in every once in awhile :)

File: interface/orders/types.php [hunk 1]
```diff
@@ -239,7 +239,7 @@ function refreshFamily(id, haskids) {
 
     // edit/add a node
     function handleNode(id, type, add, lab) {
-        var editTitle = '<i class="fa fa-pencil" style="width:20px;" aria-hidden="true"></i> ' + <?php echo xlj("Edit Mode"); ?> + ' ';
+        var editTitle = '<i class="fa fa-pencil-alt" style="width:20px;" aria-hidden="true"></i> ' + <?php echo xlj("Edit Mode"); ?> + ' ';
```
Comment (tywrenn) [inline interface/orders/types.php]: We should not be setting width in pixels here and should be using the FontAwesome width classes

Comment (stu01509) [reply interface/orders/types.php]: Hello @tywrenn 

Should I have to replace inline-style width to FontAwesome class?

**Pull request merged**